src/components/ebay-infotip/index.js

```diff
@@ -0,0 +1,44 @@
+const processHtmlAttributes = require('../../common/html-attributes');
+const emitAndFire = require('../../common/emit-and-fire');
+const template = require('./template.marko');
+
+function getInitialState(input) {
+    const iconTag = input.iconTag;
```

Comment:
I would prefer that we return a new object here instead of mutating input. By returning an object it is clear which state is actually going to be passed to the template and this would make it consistent with the rest of the components in the repo. Mutating the input can also cause undesired effects when rerendering.

Reply:
We are using `Object.assign` elsewhere, so we can use that here also.